.github/workflows/continuous-integration-workflow.yml

```diff
@@ -1,210 +1,120 @@
+
+  jenkins-agent-maven-ubi8:
+    name: Jenkins agent Maven (UBI8)
+    runs-on: ubuntu-18.04
+    steps:
+      -
+        name: Checkout repository
+        uses: actions/checkout@v2.0.0
+      -
+        name: Build docker image
+        working-directory: common/jenkins-agents/maven/docker
+        run: |
+            docker build --tag agent-maven-test-ubi8 --file Dockerfile.ubi8 \
+              --build-arg nexusUrl=https://nexus.example.com \
+              --build-arg nexusUsername=developer \
+              --build-arg nexusPassword=s3cr3t \
+              .
+
+  jenkins-agent-nodejs12-rhel7:
+    name: Jenkins agent NodeJS 12 (RHEL7)
+    runs-on: ubuntu-18.04
+    steps:
+      -
+        name: Checkout repository
+        uses: actions/checkout@v2.0.0
+      -
+        name: Build docker image
+        working-directory: common/jenkins-agents/nodejs12/docker
+        run: |
+            docker build --tag agent-nodejs12-test-rhel7 --file Dockerfile.rhel7 \
+              --build-arg nexusUrl=https://nexus.example.com \
+              --build-arg nexusAuth=developer:s3cr3t \
+              .
+
+  jenkins-agent-nodejs12-ubi8:
+    name: Jenkins agent NodeJS 12 (UBI8)
+    runs-on: ubuntu-18.04
+    steps:
+      -
+        name: Checkout repository
+        uses: actions/checkout@v2.0.0
+      -
+        name: Build docker image
+        working-directory: common/jenkins-agents/nodejs12/docker
+        run: |
+            docker build --tag agent-nodejs12-test-ubi8 --file Dockerfile.ubi8 \
+              --build-arg nexusUrl=https://nexus.example.com \
+              --build-arg nexusAuth=developer:s3cr3t \
+              .
+
+  jenkins-agent-python-rhel7:
+    name: Jenkins agent Python (RHEL7)
+    runs-on: ubuntu-18.04
+    steps:
+      -
+        name: Checkout repository
+        uses: actions/checkout@v2.0.0
+      -
+        name: Build docker image
+        working-directory: common/jenkins-agents/python/docker
+        run: |
+          docker build --tag agent-python-test-rhel7 --file Dockerfile.rhel7 .
+
+  jenkins-agent-python-ubi8:
+    name: Jenkins agent Python (UBI8)
+    runs-on: ubuntu-18.04
+    steps:
+      -
+        name: Checkout repository
+        uses: actions/checkout@v2.0.0
+      -
+        name: Build docker image
+        working-directory: common/jenkins-agents/python/docker
+        run: |
+          docker build --tag agent-python-test-ubi8 --file Dockerfile.ubi8 .
+
+# jenkins-agent-scala:
+#   name: Jenkins agent Scala
+#   runs-on: ubuntu-18.04
+#   steps:
+#     -
+#       name: Checkout repository
+#       uses: actions/checkout@v2.0.0
+#     -
+#       name: Build docker image
+#       working-directory: common/jenkins-agents/scala/docker
+#       run: |
+#           docker build --tag agent-scala-test \
+#             --build-arg nexusUrl=https://nexus.example.com \
+#             --build-arg nexusUsername=developer \
+#             --build-arg nexusPassword=s3cr3t \
+#             .
```

Comment:
Changes due to CRLF -> LF, and then I used that opportunity to also removed the commented `test` job which is no longer in use, and unlikely to be in use anytime soon.